src/main/scala/akka/contrib/datareplication/ORMultiMap.scala

```diff
@@ -66,7 +66,7 @@ case class ORMultiMap private (private[akka] val map: ORMap)
   /**
    * Convenience for put. Requires an implicit Cluster.
    */
-  def +(entry: (String, Set[Any]))(implicit node: Cluster): ORMultiMap = {
+  def +[T](entry: (String, Set[T]))(implicit node: Cluster): ORMultiMap = {
```

Comment:
I guess this semantically the same thing, but just avoiding the `Any` type right?